transaction.go

```diff
@@ -22,7 +25,7 @@ func (t *tx) Rollback() error {
 	}
 
 	t.c.tx = false
-	_, err := t.c.Exec("ROLLBACK", nil)
+	_, err := t.c.ExecContext(t.ctx, "ROLLBACK", nil)
 	t.c = nil
```

Comment:
I think this should use `context.Background`, else the `ROLLBACK` won't happen when the `ctx` is cancelled. The docs for [ConnBeginTx](https://pkg.go.dev/database/sql/driver#ConnBeginTx) indicate that it runs rollback on context cancellation.

I wonder if it should even keep the `ctx` in the TX at all? In the `database/sql` code, it looks like it doesn't use it for exec/query calls. Looks like in https://github.com/lib/pq, it doesn't track the ctx in the transaction.

Reply:
makes sense, will remove ctx from Tx